packages/web-react/src/components/Alert/README.md

```diff
@@ -44,6 +44,9 @@ and [escape hatches][readme-escape-hatches].
 
 For detailed information see [Alert][alert] component.
 
+⚠️ Please pay attention to the accessibility setting when Alert is dynamically displayed. In cases where you need to use the component as an information that requires the user's immediate attention,
+you can set `role="alert"` as an [additional attributes][readme-additional-attributes].
```

Comment:
👍🏻 

```suggestion
⚠️ Please pay attention to the accessibility setting when Alert is dynamically displayed. In case you need to use the component as an information that requires the user's immediate attention,
you can set `role="alert"` as an [additional attribute][readme-additional-attributes].
```